filters/f_decoder_wrapper.c

```diff
@@ -450,8 +450,10 @@ static bool reinit_decoder(struct priv *p)
         p->decoder = driver->create(p->decf, p->codec, sel->decoder);
         if (p->decoder) {
             pthread_mutex_lock(&p->cache_lock);
-            p->decoder_desc =
-                talloc_asprintf(p, "%s (%s)", sel->decoder, sel->desc);
+            if (sel->desc && sel->desc[0])
+                p->decoder_desc = talloc_strdup(p, sel->desc);
+            else
+                p->decoder_desc = talloc_strdup(p, sel->decoder);
```

Comment:
Mpv style guide requires `{ }` for `if/else`

Reply:
```suggestion
            const char *desc = sel->desc && sel->desc[0] ? sel->desc : sel->decoder;
            p->decoder_desc = talloc_strdup(p, desc);
```

Reply:
fixed